integration_test_project/tests/assert_no_queries_are_dropped.sql

```diff
@@ -19,4 +14,4 @@ where
         a.query_id is null
         or b.query_id is null
     )
-    and date(coalesce(a.start_time, b.start_time)) < max_date.latest_start_time
+    and date(coalesce(a.start_time, b.start_time)) < current_date
```

Comment:
this fails because `query_history` contains queries that ended more recently than what `cost_per_query` contains. `cost_per_query` filters to queries that completed in the latest hour we have data in `warehouse_metering_history`

Reply:
Okay this didn't fix it. `organization_usage.rate_sheet_daily` in our CI account doesn't have records for every day:

```sql
select
    date,
    usage_type, 
    effective_rate,
    service_type
from snowflake.organization_usage.rate_sheet_daily
order by date
```

missing dates in red

![CleanShot 2022-10-14 at 18 37 28@2x](https://user-images.githubusercontent.com/11096727/195953736-5df6550a-be68-4104-aa82-40c443ea89ec.png)


I'm guessing if you don't have any compute on that day, your credit rates dont' appear. very annoying.

We'll have to adjust this logic to fill the missing rates for ALL days, not just the current date.

https://github.com/get-select/dbt-snowflake-monitoring/blob/main/models/cost_per_query.sql#L31-L45